pkg/reco/workflow.go

```diff
@@ -168,3 +169,13 @@ func pickSafestPolicy(p1, p2 *Policy) *Policy {
 		return p2
 	}
 }
+
+func transformTargetRecoConfig(targetRecoConfig *v1alpha1.HPAConfiguration) *v1alpha1.HPAConfiguration {
+	maxReplicas := targetRecoConfig.Max
+	minReplicas := targetRecoConfig.Min
+	optimalTargetUtil := targetRecoConfig.TargetMetricValue
```

Comment:
You can avoid using this additional local var. Instead use this directly in the return value.

Reply:
sure